recipe/meta.yaml

```diff
@@ -19,19 +19,21 @@ requirements:
     - {{ compiler('c') }}
   host:
     - pip
-    - python
+    - python >=3.8
     - numpy >=1.14.3
     - cython
   run:
-    - python
+    - python >=3.8
     - {{ pin_compatible('numpy') }}
     - scipy >=0.16.0
-    - scikit-learn >=0.18.1
+    - scikit-learn >=0.22.1
     - pandas >=0.17.0
-    - dask >=1.0.0
+    - dask >=2.5.0
     - toolz
     - gatspy >=0.3.0
     - cloudpickle
+    - joblib>=0.14.1
+    - setuptools-scm>=7.1.0
```

Comment:
This should go into `host` not `run`. Also need a space between the package name and the version restriction.